docs/src/content/docs/commands/PTTL.md

```diff
@@ -1,87 +1,83 @@
 
 ## Syntax
 
-```plaintext
+```
 PTTL key
 ```
 
 ## Parameters
 
-- `key`: The key for which the remaining TTL is to be retrieved. This parameter is mandatory and must be a valid key in the DiceDB database.
+| Parameter       | Description                                                              | Type    | Required |
+|-----------------|--------------------------------------------------------------------------|---------|----------|
+| `key`           | The key for which the remaining TTL is to be retrieved.                  | String  | Yes      |
 
-## Return Value
+## Return values
 
-The `PTTL` command returns an integer value representing the remaining time to live of the key in milliseconds. The possible return values are:
+| Condition                                                  | Return Value      |
+|------------------------------------------------------------|-------------------|
+| Command is successful                                      | Returns a positive integer value representing the remaining time to live of the key in milliseconds |
+| The key exists but has no associated expiration            | `-1`              |
+| The key does not exist                                     | `-2`              |
 
-- A positive integer: The remaining TTL in milliseconds.
-- `-1`: The key exists but has no associated expiration.
-- `-2`: The key does not exist.
 
 ## Behaviour
 
-When the `PTTL` command is executed, DiceDB checks the specified key to determine its remaining TTL. The command will return the TTL in milliseconds if the key exists and has an expiration. If the key exists but does not have an expiration, the command will return `-1`. If the key does not exist, the command will return `-2`.
-
-## Error Handling
+- The command is non-destructive and does not modify the key or its expiration in any way.
+- If a key's TTL is modified (e.g., by `EXPIRE` or `PEXPIRE` commands), subsequent `PTTL` calls will reflect the updated remaining time.
 
-The `PTTL` command can raise errors in the following scenarios:
 
-1. `Wrong number of arguments`: If the command is called without the required number of arguments, DiceDB will return an error.
+## Errors
 
-   - `Error Message`: `ERR wrong number of arguments for 'pttl' command`
+1. `Wrong number of arguments`:
 
-2. `Invalid key type`: If the key is not a string, DiceDB will return an error.
+   - Error Message: `(error) ERR wrong number of arguments for 'pttl' command`
+   - Occures when attempting to use this command without any arguments or with more than one argument.
 
-   - `Error Message`: `WRONGTYPE Operation against a key holding the wrong kind of value`
 
 ## Example Usage
 
-### Example 1: Key with TTL
-
-```plaintext
-SET mykey "Hello"
-PEXPIRE mykey 5000
-PTTL mykey
-```
+### Example 1: Key with Expiration
 
-`Output:`
-
-```plaintext
+```bash
+127.0.0.1:7379> SET mykey "Hello"
+127.0.0.1:7379> EXPIRE mykey 5000
```

Comment:
Hey, i think we need to change the value here as according to this command we are setting an expiry of 5000 seconds. Please cross check once.

Reply:
Indeed, thanks for poinitng it out. I have now updated it to say 10 seconds.